gcloud/storage/blob.py

```diff
@@ -838,3 +908,21 @@ def __init__(self, bucket_name, object_name):
         self.query_params = {'name': object_name}
         self._bucket_name = bucket_name
         self._relative_path = ''
+
+
+def set_customer_encryption_headers(key, headers):
```

Comment:
That's because the name is to long. `_set_customer_encryption_headers` has 32 chars and the default is `function-rgx=[a-z_][a-z0-9_]{2,30}$`

We increased the max length to 35 for methods, but not functions.